hathor/cli/run_node.py

```diff
@@ -27,6 +29,18 @@
     from hathor.cli.run_node_args import RunNodeArgs
 
 
+@contextmanager
+def temp_fifo(filename: str, tempdir: str | None) -> Iterator[str]:
+    """Context Manager for creating named pipes."""
+    os.mkfifo(filename, mode=0o666)
+    try:
+        yield filename
```

Comment:
It seems this result isn't being used. Should we move the opening of the file to here too? And use it like:

```python
with temp_fifo(path, dir) as fp:
    fp.readlines()
    # ...
```

Reply:
Done!